### PR TITLE
Dtspo 5994 add tolerations admin apps misc sbox test

### PR DESCRIPTION
**Pull request by nickmoyni**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-5994

### Change description ###

SBOX toleration testing for aad and env apps. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/admin/aad-pod-id/aad-pod-id-patch.yaml [hunk 1]
```diff
@@ -22,4 +22,4 @@ spec:
       - name: mic
         env:
         - name: FORCENAMESPACED
-          value: "true"
\ No newline at end of file
+          value: "true"
```
Comment (nickmoyni) [inline apps/admin/aad-pod-id/aad-pod-id-patch.yaml:25]: LG

**Comment by thomast1906** [comment]
looks like this needs kustomized

**Comment by nickmoyni** [comment]
Added kustomize file.

**Comment by thomast1906** [comment]
As discussed, to review the format of file. 
- include tolerations inside deployment

File: apps/admin/sbox/00/env-injector.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: env-injector-webhook
+  namespace: admin
+spec:
+  releaseName: env-injector-webhook
+  chart:
+    spec:
+      chart: env-injector-webhook
```
Comment (thomast1906) [inline apps/admin/sbox/00/env-injector.yaml:10]: Part duplication on this, only include what you require in terps of spec.*

File: apps/admin/sbox/00/aad-pod-id-patch.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+spec:
+  tolerations:
+  - effect: NoSchedule
+    key: CriticalAddonsOnly
+    operator: Equal
+    value: "true"
+  affinity:
+  nodeAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+    - weight: 1
```
Comment (thomast1906) [inline apps/admin/sbox/00/aad-pod-id-patch.yaml:14]: Lets see if this works, can we discuss this? (I did ask for this nmi to be added :D )

I am unsure if we want DaemonSet included - @hannah38 lets all catchup tomorrow.

**Review by thomast1906 — APPROVED** [review]

**Review by thomast1906 — COMMENTED** [review]
is it not .spec.values?

**Review by thomast1906 — CHANGES_REQUESTED** [review]
is it not .spec.values.*? - edited, its fine

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**